### PR TITLE
#3424 VS hang on exit

### PR DESCRIPTION
**Pull request by MikhailArkhipov**

### CODE REVIEW

File: src/R/Support/Test/RD/Help/GetFunctionInfoTest.cs [hunk 1]
```diff
@@ -70,6 +70,18 @@ public class GetFunctionInfoTest {
 
         [Test]
         [Category.R.Signatures]
+        public void GetRdFunctionArgumentsTest03() {
```
Comment (AlexanderSher) [inline src/R/Support/Test/RD/Help/GetFunctionInfoTest.cs]: Do you want to add any tests for parser itself?

Comment (MikhailArkhipov) [inline src/R/Support/Test/RD/Help/GetFunctionInfoTest.cs]: It is basically covered by quick info and signature tests.

**Comment by AlexanderSher** [comment]
👍

File: src/R/Support/Test/RD/Help/GetFunctionInfoTest.cs (hunk 1 above)
Comment (MikhailArkhipov) [inline src/R/Support/Test/RD/Help/GetFunctionInfoTest.cs]: But i guess i could add some more granular going forward

**Pull request merged**